views/newsletter/templates/components/save.hbs

```diff
@@ -1,10 +1,13 @@
 <div class="mailpoet_save_wrapper {{ wrapperClass }}">
-    <div class="mailpoet_button_group">
+    <input type="button" name="next" value="<%= __('Next') %>" class="button button-primary mailpoet_save_next float-right" />
+    <div class="mailpoet_button_group float-right">
         <input type="button" name="save" value="<%= __('Save') %>" class="button button-primary mailpoet_save_button" /><button class="button button-primary mailpoet_save_show_options" data-automation-id="newsletter_save_options_toggle" ><span class="dashicons mailpoet_save_show_options_icon"></span></button>
     </div>
-    <input type="button" name="next" value="<%= __('Next') %>" class="button button-primary mailpoet_save_next" />
-    <span class="mailpoet_save_error"></span>
-    <div class="mailpoet_editor_last_saved mailpoet_hidden"><%= __('Autosaved') %> <span class="mailpoet_autosaved_at"></span></div>
+    <div class="clearfix"></div>
+    <div class="mailpoet_editor_messages">    
```

Comment:
It looks good, but because these containers are turned into floats, it seems to collapse the bottom margin that was applied to the `#mailpoet_editor_bottom` element.
The bottom margin was applied so that there is some extra space between the bottom of the editor and bottom of the page, allowing the save button, any error messages, autosave to be displayed without the Beacon button displaying over it.

E.g. see how it was before in the original ticket: https://d.pr/i/ylExOn
Note that "Autosave" text bottom right of the image is fully visible, and the Beacon is displayed underneath it, without occluding it.

Now in the current situation several elements are obstructed, making it difficult to see or interact with them:
![screenshot from 2018-11-07 11-45-39](https://user-images.githubusercontent.com/562769/48123933-cf38cf00-e283-11e8-97a1-fd1bdba0bc02.png)

So we need to find a way to avoid the Beacon obstructing other elements.